feeadjuster/feeadjuster.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 import random
 import time
-from pyln.client import Plugin, RpcError
+from pyln.client import Plugin, Millisatoshi, RpcError
```

Comment:
Err, i was reluctant to using Millisatoshi, it's always been a source of bugs in my experience... Hopefully m-schmoock is on the patch of trackin'em all down!